src/Microsoft.ML.Dnn/ImageClassificationTransform.cs

```diff
@@ -1065,6 +1184,12 @@ internal sealed class Options : TransformInputBase
             [Argument(ArgumentType.AtMostOnce, HelpText = "Learning rate to use during optimization.", SortOrder = 12)]
             public float LearningRate = 0.01f;
 
+            /// <summary>
+            /// Early Stopping technique to stop training when accuracy stops improving.
+            /// </summary>
+            [Argument(ArgumentType.AtMostOnce, HelpText = "Early Stopping technique to stop training when accuracy stops improving.", SortOrder = 15)]
+            public EarlyStopping EarlyStopper;
```

Comment:
>EarlyStopper [](start = 33, length = 12)

We generally refer this as EarlyStoppingCriteria

Reply:
renamed

---
In reply to: [330318020](https://github.com/dotnet/machinelearning/pull/4237#discussion_r330318020) [](ancestors = 330318020)